common/utils.js

```diff
@@ -785,6 +785,39 @@ function validateDestinationChain(chains, destinationChain) {
     validateChain(chains, destinationChain);
 }
 
+async function estimateITSFee(chain, destinationChain, env, eventType, gasValue, gmpAxelarscanApi) {
+    if (!gmpAxelarscanApi || env === 'devnet-amplifier') {
+        printInfo('Using default ITS fee: 0');
+        return 0;
+    }
+
+    if (isValidNumber(gasValue)) {
```

Comment:
Also check if it's set to auto and fail otherwise

Reply:
I just mean handling the scenario where it's not a number or set to 'auto'. That's a bad input so we should fail validation (you can validate it in commander CLI definition probably too)